core/src/main/java/org/jdbi/v3/core/statement/Slf4JSqlLogger.java

```diff
@@ -0,0 +1,69 @@
+package org.jdbi.v3.core.statement;
+
+import java.time.Duration;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Simple {@link SqlLogger} that emits some diagnostic information about
+ * Jdbi usage.  Usually you shouldn't log every database statement made but
+ * it can be helpful for debugging.
+ */
+public class Slf4JSqlLogger implements SqlLogger {
+    private final Logger log;
+
+    public Slf4JSqlLogger() {
+        this(LoggerFactory.getLogger("org.jdbi.sql"));
+    }
+
+    public Slf4JSqlLogger(Logger log) {
+        this.log = log;
+    }
+
+    @Override
+    public void logAfterExecution(StatementContext context) {
+        if (log.isDebugEnabled()) {
+            log.debug("Executed in {} '{}' with parameters '{}'",
+                    fmt(Duration.between(context.getExecutionMoment(), context.getCompletionMoment())),
+                    context.getParsedSql().getSql(),
+                    context.getBinding());
+        }
+    }
+
+    private String fmt(Duration dur) {
```

Comment:
I'd rather not abbreviate method names and params or variables (except the h/m/s, those are ubiquitously known), especially ts is cryptical